src/fielddef.ts

```diff
@@ -374,6 +375,12 @@ export function getFieldDef<F>(channelDef: ChannelDef<F>): FieldDef<F> {
  * Convert type to full, lowercase type, or augment the fieldDef with a default type if missing.
  */
 export function normalize(channelDef: ChannelDef<string>, channel: Channel): ChannelDef<any> {
+  if (isString(channelDef) || isNumber(channelDef)) {
+    const type = isString(channelDef) ? 'string' : 'number';
```

Comment:
What about booleans?

Reply:
Not sure if boolean makes sense for any channel.  But if you think I should add I can add. Let me know.